tests/test_appclass.py

```diff
@@ -0,0 +1,40 @@
+import aiida
+import pytest
+import traitlets
+from packaging import version
+
+from aiidalab.app import AiidaLabApp
+
+
+def test_init_refresh(generate_app):
+    app = generate_app()
+    assert len(app.available_versions) == 0
+    # After refresh the availale_versions traitlets is updated
+    app.refresh()
+    assert len(app.available_versions) != 0
+
+
+def test_prereleases(generate_app):
+    app = generate_app()
+
+    # without prereleases tick
+    app.refresh()
+    assert app.has_prereleases
+    assert app.include_prereleases is False
+
+    # tick prereleases tick
+    app.include_prereleases = True
+    assert "v23.01.0b1" in app.available_versions
```

Comment:
The negative of this assert should be few lines above before the `include_prereleases` is set to True

Reply:
@danielhollas You are asking for the changes or it is a comment we can add?

Reply:
I am asking to add an extra assert above line 24 which will test that the pre-release version is not included when the "include_prerelease" is not ticked.

```python
assert "v23.01.0b1 not in app.available_versions
```